test/plots/wheelPlotTests.ts

```diff
@@ -0,0 +1,304 @@
+          "There should be exactly 3 entities in the wheel chart, representing the valid values");
+
+        svg.remove();
+      });
+    });
+
+    describe("innerRadius and outerRadius", () => {
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
+      let svg: d3.Selection<void>;
+      let wheelPlot: Plottable.Plots.Wheel<number, number>;
+      let rScale: Plottable.Scales.Linear;
+      let tScale: Plottable.Scales.Linear;
+      let data: [any];
+      let dataset: Plottable.Dataset;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        rScale = new Plottable.Scales.Linear();
+        tScale = new Plottable.Scales.Linear();
+        tScale.domain([0, 360]);
+        wheelPlot = new Plottable.Plots.Wheel();
+        wheelPlot.startAngle((d) => d.t1, tScale);
+        wheelPlot.endAngle((d) => d.t2, tScale);
+        data = [
+          {r1: 0, r2: 1, t1: 0, t2: 180 },
+          {r1: 1, r2: 2, t1: 180, t2: 360 }];
+        dataset = new Plottable.Dataset(data);
+        wheelPlot.addDataset(dataset);
+      });
+
+      it("can set and get innerRadius", () => {
+        wheelPlot.outerRadius((d) => d.r2, rScale);
+        assert.isUndefined(wheelPlot.innerRadius(), "innerRadius is initiized to undefined");
+
+        wheelPlot.innerRadius(0);
+        wheelPlot.renderTo(svg);
+        let constInnerRadiusAccessor = (<any> Plottable.Plot)._scaledAccessor(wheelPlot.innerRadius());
+        assert.strictEqual(constInnerRadiusAccessor(data[0], 0, dataset), 0, "access innerRadius that is set to a constant");
+        assert.strictEqual(constInnerRadiusAccessor(data[1], 1, dataset), 0, "access innerRadius that is set to a constant");
+
+        wheelPlot.innerRadius((d: any) => d.r1);
+        let innerRadiusAccessor = (<any> Plottable.Plot)._scaledAccessor(wheelPlot.innerRadius());
+        assert.strictEqual(innerRadiusAccessor(data[0], 0, dataset), 0, "access innerRadius correctly without a scale");
+        assert.strictEqual(innerRadiusAccessor(data[1], 1, dataset), 1, "access innerRadius correctly without a scale");
+
+        wheelPlot.innerRadius((d: any) => d.r1, rScale);
+        let scaledInnerRadiusAccessor = (<any> Plottable.Plot)._scaledAccessor(wheelPlot.innerRadius());
+        assert.strictEqual(scaledInnerRadiusAccessor(data[0], 0, dataset), rScale.scale(0), "access innerRadius correctly with a scale");
+        assert.strictEqual(scaledInnerRadiusAccessor(data[1], 1, dataset), rScale.scale(1), "access innerRadius correctly with a scale");
+
+        svg.remove();
+      });
+
+      it("can set and get outerRadius", () => {
+        wheelPlot.innerRadius((d) => d.r1, rScale);
+        assert.isUndefined(wheelPlot.outerRadius(), "outerRadius is initiized to undefined");
+
+        wheelPlot.outerRadius(0);
+        wheelPlot.renderTo(svg);
+        let constEndAngleAccessor = (<any> Plottable.Plot)._scaledAccessor(wheelPlot.outerRadius());
```

Comment:
Same feedback here -- this test doesn't actually check the getter mode of `outerRadius()`